src/libs/ajax/DatasetBuilder.test.ts

```diff
@@ -62,15 +60,14 @@ const rangeOption: ProgramDataRangeOption = {
   kind: 'range',
   min: 0,
   max: 101,
-  id: 3,
   name: 'rangeOption',
 };
 
 const rangeCriteria: ProgramDataRangeCriteria = {
   kind: 'range',
   name: 'rangeCriteria',
   rangeOption,
-  id: 4,
+  index: 4,
```

Comment:
What does the index on a criteria represent?

Reply:
Yes, its only purpose is to support async loading of data in the UI. So it's a UI internal value that has no other meaning or use.